test/runtime/hello.cc

```diff
@@ -0,0 +1,33 @@
+// Basic tests of I/O API
```

Comment:
This is very cool.  What causes the compiled version of this to be executed?

Reply:
It's run from `test/runtime/CMakeLists.txt`.